### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by GiovanYCringe**

### CODE REVIEW

File: changelog_deen.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Highlights & Device Specific Changes:
+Build type: Beta
+Device: Motorola one (deen)
+Device maintainer: GiovanYCringe
+
+===== <date> =====
```
Comment (gwolf2u) [inline changelog_deen.txt]: Add date

File: changelog_deen.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Forceencrypt enable by default
```
Comment (gwolf2u) [inline changelog_deen.txt:7]: Maybe set "initial support"

File: deen.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "GiovanYCringe",
+        "oem": "Motorola",
+        "device": "Motorola one",
```
Comment (gwolf2u) [inline deen.json]: Try "one" with capital letter "O" not "o"

File: deen.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "filename": "crDroidAndroid-12.1-20220807-deen-v8.7.zip",
+        "download": "https://sourceforge.net/projects/giovany-cringe/files/Roms/crDroid-Deen/crDroidAndroid-12.1-20220807-deen-v8.7.zip/download",
```
Comment (gwolf2u) [inline deen.json]: Use generated download url from script

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: deen.json (hunk 3 above)
Comment (GiovanYCringe) [reply deen.json]: Done

File: deen.json (hunk 4 above)
Comment (GiovanYCringe) [reply deen.json]: done

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**